### PR TITLE
38 register client warning bug

### PR DESCRIPTION
**Pull request by dylankelly04**
- Fixed bug so now when an invalid phone number is entered only the error message below appears.
- Minor CSS change

### CODE REVIEW

**Comment by mengzhuou** [comment]
image
image

Seems like there's an issue registering with the phone even if the format is correct and phone number does not already exist. Others seem all good!

File: Frontend/frontend/src/index.css [hunk 1]
```diff
@@ -52,7 +52,7 @@
 body {
   font-family: "Nunito", sans-serif;
   min-height: 100vh;
-  background-color: var(--BGCOLOR);
+  /* background-color: var(--BGCOLOR); */
```
Comment (mengzhuou) [inline Frontend/frontend/src/index.css]: Is there a reason why you comment this out?

Comment (dylankelly04) [reply Frontend/frontend/src/index.css]: It's just the current color is is hard on the eyes, this is what it looks like now:
Screenshot 2023-10-30 at 21 34 41
Not very important change could definately just remove that if needed

**Comment by mengzhuou** [comment]
If you place a registered primary phone number, and a non-registered secondary phone number and submit, it should not clear second phone number text-field. Instead, it should clear whichever is having an issue.
image

**Review by mengzhuou — APPROVED** [review]

**Pull request merged**